components/retro-go/rg_system.h

```diff
@@ -18,21 +32,20 @@ extern "C" {
     #include "targets/qtpy-gamer.h"
 #elif defined(RG_TARGET_RETRO_ESP32)
     #include "targets/retro-esp32.h"
+#elif defined(RG_TARGET_ESPLAY_S3)
+	#include "targets/esplay-s3.h"
 #else
     #warning "No target defined. Defaulting to ODROID-GO."
     #include "targets/odroid-go.h"
     #define RG_TARGET_ODROID_GO
 #endif
 
+#define RG_APP_MFW "mfw_app"
 #define RG_APP_LAUNCHER "launcher"
 #define RG_APP_FACTORY  NULL
 
 #define RG_PATH_MAX 255
 
-// This is the base task priority used for system tasks.
```

Comment:
This should be undone, I think you didn't pull changes I've made before pushing.